rps.js

```diff
@@ -0,0 +1,46 @@
+
+var Decisions = function() {
+    userChoice = prompt("Do you choose rock, paper or scissors?");
+    computerChoice = Math.random();
+    if (computerChoice < 0.4) {
+    computerChoice = "rock";
+    } else if(computerChoice <= 0.8) {
+    computerChoice = "paper";
+    } else {
+    computerChoice = "scissors";
+    }
+    console.log("User: " + userChoice);
+    console.log("Computer: " + computerChoice);
+    }
+    
+    Decisions();
+    
+    var compare = function(choice1, choice2) {
```

Comment:
the logic doesn't work for your program, the computer will always play the same move as the player.